packages/path_provider/path_provider/lib/path_provider.dart

```diff
@@ -128,7 +128,7 @@ Future<Directory> getLibraryDirectory() async {
 /// On iOS, this uses the `NSDocumentDirectory` API. Consider using
 /// [getApplicationSupportDirectory] instead if the data is not user-generated.
 ///
-/// On Android, this uses the `getDataDirectory` API on the context. Consider
+/// On Android, this uses the `getDir` API on the context. Consider
```

Comment:
> Thank you for your reply, you are right, we cannot guarantee whether the engine implementation will change frequently.

I'm glad we've reached agreement, but you have not actually addressed my feedback about changing this comment accordingly.

Reply:
Here are general instructions for this method:   
https://github.com/PollexLee/plugins/blob/aa242b45a1c50c212a4e85e8f4def67ee4f7384d/packages/path_provider/path_provider/lib/path_provider.dart#L125

I think the implementation method of the native side in the comment is to facilitate the development of native programmers.

About how to fix the anti-pattern, i have no idea. 😂

Reply:
As I said in my initial review:

> this should be changed to say something more generic about it using application-scoped storage.

Reply:
Are you planning on updating the PR to address this feedback?

Reply:
Sorry, I am not sure how to address this feedback.

Reply:
I'm saying that instead of saying "this uses the `getDir` API on the context", it should say something like "this uses application-scoped storage". That provides context for the recommendation in the next sentence, without tying it to implementation details that aren't actually controlled by the plugin.